### PR TITLE
Try instrumenting Solr queries for better profiling

### PR DESCRIPTION
**Pull request by mbarnett**
If I've squinted at the Skylight documentation and an implementation I found in the netflix/fast_jsonapi repo correctly, this should give us some insight into what queries are being run against solr, in Skylight.

### CODE REVIEW

File: config/initializers/solr_normalizer.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require 'skylight'
+
+SOLR_NORMALIZER = 'solr.query'.freeze
+JUPITER_SOLR_NOTIFIFCATION = "jupiter.#{SOLR_NORMALIZER}".freeze
```
Comment (pbinkley) [inline config/initializers/solr_normalizer.rb]: typo: notifi*f*cation

File: config/initializers/solr_normalizer.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+class Skylight::Normalizers::SolrNormalizer < Skylight::Normalizers::Normalizer
+
+  register JUPITER_SOLR_NOTIFIFCATION
```
Comment (pbinkley) [inline config/initializers/solr_normalizer.rb]: or was that intentional?

File: test/models/jupiter_core/search_test.rb [hunk 3]
```diff
@@ -76,7 +76,20 @@ def unlocked_method_dont_let_locked_methods_mutate(attempted_title)
       end
     end
 
-    search_results = JupiterCore::Search.faceted_search(models: @@klass, q: creator2)
+    # ensure searches are sending notifications
+    events = []
+    ActiveSupport::Notifications.subscribe(JUPITER_SOLR_NOTIFIFCATION) do |*args|
```
Comment (pbinkley) [inline test/models/jupiter_core/search_test.rb]: last one

File: config/initializers/solr_normalizer.rb (hunk 1 above)
Comment (mbarnett) [reply config/initializers/solr_normalizer.rb]: No no that's a totally real word I DEFINITELY didn't typo. It means a notification that MIGHT happen IF something happens.

**Pull request merged**